velox/exec/RowContainer.cpp

```diff
@@ -135,20 +135,20 @@ RowContainer::RowContainer(
   // Compute the layout of the payload row.  The row has keys, null
   // flags, accumulators, dependent fields. All fields are fixed
   // width. If variable width data is referenced, this is done with
-  // StringView that inlines or points to the data.  The number of
-  // bytes used by each key is determined by keyTypes[i].  Null flags
-  // are one bit per field. If nullableKeys is true there is a null
-  // flag for each key. A null bit for each accumulator and dependent
-  // field follows.  If hasProbedFlag is true, there is an extra bit
-  // to track if the row has been selected by a hash join probe. This
-  // is followed by a free bit which is set if the row is in a free
-  // list. The accumulators come next, with size given by
-  // Aggregate::accumulatorFixedWidthSize(). Dependent fields follow.
-  // These are non-key columns for hash join or order by. If there are variable
-  // length columns or accumulators, i.e. ones that allocate extra space, this
-  // space is tracked by a uint32_t after the dependent columns. If this is a
-  // hash join build side, the pointer to the next row with the same key is
-  // after the optional row size.
+  // StringView(for VARCHAR) and std::string_view(ARRAY, MAP and ROW) that
+  // inlines or points to the data.  The number of bytes used by each key is
```

Comment:
> that inlines or points to the data

This applies only to VARCHAR, not to complex types, no?

Reply:
Yes, std::string_view won't do inlining.
I've fixed, thanks!